pkg/models/shortUrl.go

```diff
@@ -0,0 +1,47 @@
+package models
+
+import (
+	"errors"
+)
+
+var (
+	ErrShortUrlNotFound = errors.New("Short URL not found")
+)
+
+type ShortUrl struct {
+	Id         int64
+	OrgId      int64
+	Uid        string
+	Path       string
+	CreatedBy  int64
+	CreatedAt  int64
+	LastSeenAt int64
+}
+
+// ---------------------
+// COMMANDS
+
+type CreateShortUrlCommand struct {
+	OrgId     int64  `json:"orgId"`
+	Uid       string `json:"uid"`
+	Path      string `json:"path"`
+	CreatedBy int64  `json:"userId"`
```

Comment:
We already have the contextual user id in the backend and we don't want this to be overridden from client.

```suggestion
	CreatedBy int64  json:"-"`
```